deploy/kubernetes/deploy-kiali-to-kubernetes.sh

```diff
@@ -228,3 +228,23 @@ do
     exit 1
   fi
 done
+
+# Deploy Kiali MonitoringDashboards to kubernetes
+# Note for undeploy script: dashboards are implicitely undeployed when the related CRD is removed
+echo "Deploying Kiali dashboards to Kubernetes project ${NAMESPACE}"
```

Comment:
There is no concept of "project" in k8s, can you replace this "project" with namespace - and do it in the other spot in the code you didn't change? :) We should say deploying to Kubernetes "namespace"